src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -37,7 +37,7 @@ file that was distributed with this source code.
         {% endif -%}
     </div>
 
-    {% if sonata_admin.field_description.hasAssociationAdmin %}
+    {% if sonata_admin.field_description.hasAssociationAdmin is defined %}
```

Comment:
Youu should add check for `sonata_admin.field_description` and revert this change